<ui version="4.0" >
 <class>Preferences</class>
 <widget class="QDialog" name="Preferences" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>117</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>icons/cog.png</normaloff>icons/cog.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Host Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QSpinBox" name="slotBox" />
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="slotLabel" >
        <property name="text" >
         <string>Slots</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="textLabel4" >
        <property name="text" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p align="justify" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Set the number of slots (CPU cores)&lt;/span>&lt;/p>
&lt;p align="justify" style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">you are able to spare&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
